docs/sphinx/cookbook/dynamic_pool_heuristics.rst

```diff
@@ -0,0 +1,43 @@
+.. _dynamic_pool_heuristics:
+
+=======================================
+Creating a Coalesce Heurisitic Function
+=======================================
+
+By default, a DynamicPool object will not coalesce blocks that it allocates
```

Comment:
Could define what you mean by coalesce

Reply:
FYI - working on this now...

Reply:
Okay, @davidbeckingsale, please take another look.